package.json

```diff
@@ -21,7 +21,7 @@
     "react-dom": "^16.11.0"
   },
   "scripts": {
-    "install": "npm run bootstrap",
+    "install": "npm install lerna && npm run bootstrap",
```

Comment:
Instead I would add `lerna` as a dependency, keep this script the same

Reply:
Another approach might be to remove `install` altogether. This trips netlify. Do we need it for something?

Reply:
Yes, I think that we still need to cause `install` to actually trigger `lerna bootstrap`

Reply:
> Yes, I think that we still need to cause `install` to actually trigger `lerna bootstrap`

I fixed the netlify deploy to do `npm install -g lerna && lerna bootstrap && npm run build` and it seems to work!

Reply:
by default netlify calls `npm install` if present.

Reply:
But without an `install` script, netlify won't know to call `lerna bootstrap`, right?

Reply:
This seems to work:

<img width="916" alt="Screenshot 2020-02-24 at 9 46 12 AM" src="https://user-images.githubusercontent.com/670099/75161874-84e29800-56ea-11ea-998e-4457ccb8ddc1.png">

It was already there so I assumed it was your idea in the first place? It was just not working with `install` which is called before the build command.

Reply:
oh, sorry
! It's all good then!

Reply:
Proof is in the pudding: netlify link is below in this PR.